.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: continuous-integration
+on: [pull_request]
+jobs:
+  test-runner:
+    runs-on: ubuntu-16.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

This would do ig @HarshKhandeparkar

Reply:
Great!

Reply:
but shouldn't we try to freeze version? otherwise an ubuntu upgrade might break our build... unlikely but possible... but open to either...

Reply:
@jywarren yeah, we reverted to Ubuntu 18.04 in the next commit we Ubuntu 20.04 had broke the ci.